docs/src/components/egghead.stories.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { Egghead } from '../../../@pauliescanlon/gatsby-mdx-embed/src/components/Egghead'
+
+<Meta title="Components/Egghead" component={Egghead} />
+
+# Egghead
+
+Display an Egghead video by including the component in your `.mdx`. The `lesson` prop is required.
+
+<Props of={Egghead} />
+
+## Basic usage
+
+<Preview>
+  <Egghead lesson="gatsby-set-up-a-gatsby-site-to-use-mdx-with-gatsby-plugin-mdx-with-a-default-layout" />
```

Comment:
Would you mind adding a little something to `docs/src/a-pages/3.help.stories.mdx`
You'll see with the other i've tried to include what the url looks like and which bits you need to pass on as the prop.

Reply:
Do you have an example page?

I was comparing it to YouTube but the Egghead one is simple so there's only the basic usage, right??

![image](https://user-images.githubusercontent.com/234708/88819442-e639f600-d1b7-11ea-83c9-296ea74eb0b3.png)

Reply:
In the `help.stores.mdx` file there's a instructions for each of the components `docs/src/a-pages/3.help.stories.mdx`